### PR TITLE
mojito start output version of mojito being run

### PR DESCRIPTION
**Pull request by lzhan**
Fix trello card 219.

### CODE REVIEW

**Comment by mojit0** [comment]
+1

**Comment by lzhan** [comment]
I see. Thanks Isao. I will update.

**Comment by caridy** [comment]
@lzhan can we also remove the internal static reference to version from mojito.common.js in autoload?

```
YUI.add('mojito', function(Y, NAME) {

    Y.namespace('mojito').version = '0.2';
```

**Comment by lzhan** [comment]
Yes, I will remove it.

**Comment by drewfish** [comment]
Beware that `lib/app/mojits/HTMLFrameMojit/controller.server.js` uses `Y.mojito.version`.  I know that `lib/app/autoload/store.server.js` has the mojito version, so perhaps we can expose that somehow.  Also I think the RS isn't quite getting the mojito version correctly, so that needs to be updated.

Hmm... perhaps we could have the RS populate `Y.mojito.version`.

**Comment by lzhan** [comment]
controller.server.js does not get right mojito version: 'Powered by Mojito ' + Y.mojito.version;  Y.mojito.version gives undefined.

**Comment by drewfish** [comment]
Yes exactly, since you removed Y.mojito.version :)  I would be OK with changing HTMLFrameMojit's controller to just say "Powered by Mojito".

**Comment by lzhan** [comment]
I see. I was wondering if it is used anywhere :). I will check if RS can populate Y.mojito.version.

File: lib/app/commands/start.js [hunk 1]
```diff
@@ -10,6 +10,7 @@
 var path = require('path'),
     utils = require(path.join(__dirname, '../../management/utils')),
     Mojito = require(path.join(__dirname, '../../mojito')),
+    mojitoVersion = require('../../../package.json').version,
```
Comment (drewfish) [inline lib/app/commands/start.js]: I think you want to do the `path.join(__dirname, ...)` here too.

**Comment by drewfish** [comment]
+1

**Pull request merged**